ghost/core/core/server/services/milestone-emails/index.js

```diff
@@ -0,0 +1,57 @@
+// Stubbing stripe in test was causing issues. Moved it
```

Comment:
Can you rename this file to `service.js` and then create an `index.js` with contents of `module.exports = require('./service.js')`